test/components/form/submission.spec.js

```diff
@@ -184,4 +187,36 @@ describe('FormSubmission', () => {
           }));
     });
   });
+
+  describe('session timeout', () => {
+    beforeEach(() => {
+      mockLogin();
+    });
+
+    it('does not auto log out', async () => {
+      testData.extendedForms.createPast(1, { xmlFormId: 'a', webformsEnabled: true });
+      const session = testData.sessions.createNew({ expiresAt: '1970-01-01T00:05:00Z' });
+
+      const clock = sinon.useFakeTimers();
+
+      await load('/projects/1/forms/a/submissions/new', mountOptions({
+        container: {
+          requestData: testRequestData([], {
+            session
+          }),
```

Comment:
It feels a little awkward to call `mockLogin()`, then create a different session from the one that `mockLogin()` creates and uses. For example, `sessionExpires` in local storage will end up not matching the new session.

What do you think about modifying `mockLogin()` to make it easier to customize the session that it creates? `mockLogin()` accepts options that it will use to create the user. We could add a second parameter for options that it will use to create the session.

If we did something like that, you wouldn't need to pass `session` to `requestData`, since `mockLogin()` automatically passes the session it creates to `requestData`.

Reply:
this is not needed after I moved sinon.useFakeTimers() before everything else including mockLogin